code/modules/mob/living/carbon/resist.dm

```diff
@@ -69,12 +69,12 @@
 			if(GRAB_PASSIVE)
 				qdel(G)
 			if(GRAB_AGGRESSIVE)
-				if(prob(60)) //same chance of breaking the grab as disarm
+				if(prob(max(60 + ((stats?.getStat(STAT_ROB)) - G.assailant?.stats.getStat(STAT_ROB) ** 0.8), 1))) // same scaling as cooldown increase and if you manage to be THAT BAD, 1% for luck
```

Comment:
You would need to have 0 rob while your attacker has 160 to get down to the 1%

Reply:
that can happen with powergaming ROB.